src/test/java/com/ironcorelabs/tenantsecurity/kms/v1/DevIntegrationTest.java

```diff
@@ -192,6 +192,7 @@ public void roundTripAzureKMSTest() throws Exception {
     assertEqualBytes(decryptedValuesMap.get("doc3"), documentMap.get("doc3"));
   }
 
+  @SuppressWarnings("deprecation")
```

Comment:
Do you still need that after switching to `getSuccesses`?

Reply:
yeah, it's for our deprecated `rekeyDocument` function. Which we could choose to remove on this breaking change...

Reply:
I think we should remove it. I asked in slack as well to see if anyone disagrees.

Reply:
Removed